### PR TITLE
Add/style complaint views

### LINKED ISSUE #24: Style complaint view
# REQUIREMENTS
Apply styles to complaints views.
Add edit permissions to views.

- [x] Index
- [x] Show

### PR DESCRIPTION
**Pull request by brayandiazc**
# Description
The complaint views will be refactored, gems a were added for editing without reloading html, gem for paging and filters:

## Problems
Problems were encountered when logging in and sending the user to the root path (image is attached)
error

  
## Screenshots 
complaint-unregistred
complaint-show
complaint-index

### CODE REVIEW

**Review by ALAN1MACIAS — APPROVED** [review]

File: app/controllers/complaints_controller.rb [hunk 1]
```diff
@@ -22,7 +28,7 @@ def edit
 
   # POST /complaints or /complaints.json
   def create
-    @complaint = Complaint.new(complaint_params)
+    @complaint = current_user.complaints.build(complaint_params)
```
Comment (Brayan9105) [inline app/controllers/complaints_controller.rb:31]: good use of nested resources

**Review by Brayan9105 — APPROVED** [review]
The views looks great

**Review by jaimelr — APPROVED** [review]
Looks good 👍🏼

**Comment by jaimelr** [comment]
A lot of things are happening in this PR consider refactoring into single-purpose PRs in the future, to make code reviews easier.

**Pull request merged**